src/SFA.DAS.EAS.Portal.Client/SFA.DAS.EAS.Portal.Client.csproj

```diff
@@ -14,6 +14,8 @@
         <PublishRepositoryUrl>true</PublishRepositoryUrl>
         <IncludeSymbols>true</IncludeSymbols>
         <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+        <Version>1.7.491</Version>
```

Comment:
The version number will be injected during the package release process, so we should remove this (and also the assembly version). We can add it to create the package locally if required, but shouldn't check it in.